src/main/java/frc/robot/commands/drivetrain/drive.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.drivetrain;
+
+import java.util.function.Supplier;
+
+import com.ctre.phoenix6.mechanisms.swerve.SwerveModule.DriveRequestType;
+import com.ctre.phoenix6.mechanisms.swerve.SwerveRequest;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.constants.drivetrain.DrivetrainConstants;
+
+public class drive extends Command
+{
+  private Supplier<Double> SPEED_X_SUPPLIER, SPEED_Y_SUPPLIER, ROTATIONAL_SPEED_SUPPLIER;
```

Comment:
Need to provide documentation on any class members or variables even if they're private.

Reply:
Are these considered constants?

Reply:
How would documentation look for these variables in particular?

Reply:
Something like
/**
@brief Basic Drive Command
*/
for the class

as for variables try to avoid single line assignment and just describe what each on does.